tests/dummy/app/index.html

```diff
@@ -9,8 +9,8 @@
 
     {{content-for "head"}}
 
-    <link rel="stylesheet" href="{{rootURL}}assets/vendor.css">
-    <link rel="stylesheet" href="{{rootURL}}assets/dummy.css">
+    <link integrity="" rel="stylesheet" href="{{rootURL}}assets/vendor.css">
```

Comment:
what does this do?

Reply:
It's supposed to let the browser check to see if the code was manipulated. (I didn't know before so I googled):

> Integrity attribute is to allow the browser to check the file source to ensure that the code is never loaded if the source has been manipulated.

https://stackoverflow.com/a/32042283/1352387